### ISSUE TITLE
Update leaflet library to provide a11y improvements

### ISSUE THREAD

**Issue by kimisgold** [omeka/plugin-Geolocation#56]
An accessibility audit has flagged a number of challenges for screen readers:

* Poor labeling for the "close" buttons on individual marker panes.
* Map markers cannot be navigated to via keyboard.
* Screen readers do not announce the map when it comes into focus.

I think Geolocation is currently using Leaflet 1.6.0, and there have been several accessibility updates since that may address these issues.

**Comment by jimsafley** [comment]
The [update-leaflet-library](https://github.com/omeka/plugin-Geolocation/compare/master...update-leaflet-library) updates the Leaflet library to v1.9. Hopefully that'll address the issues.

**Comment by zerocrates** [comment]
Yes, I'd looked into this a week or so ago. The "close" button should definitely be included in this update. Not sure about the others.

**Comment by kimisgold** [comment]
I can confirm the close button issue is resolved, as well as the map markers now being navigable via keyboard. I think there are some other elements we have to do ourselves: it seems currently not possible to navigate to popup pane content once a map marker is clicked, and we need a better aria role for the map itself.

**Comment by zerocrates** [comment]
The update here is merged, along with updates to the other two leaflet-related dependencies we use.

Other things I'll note here:

- The marker clusterer did seem to have some accessibility updates, but looks like it has more (in particular aria labeling for the cluster) that were done a long time ago but haven't been released... they haven't made a release in a long time. We could pull it in manually if necessary.
- Leaflet allows/recommends setting an alt for markers which we're not currently doing, markers have an alt of "Marker" by default it looks like

**Comment by kimisgold** [comment]
I've provided the item title as an `alt` for the markers. (28dfc6a36ae859d1d411b2a4329f91327f1cdc70).

I think I'd be okay with the manual updates getting pulled in. We could always upgrade again whenever they do a formal release including any future fixes.

**Comment by jimsafley** [comment]
I think this is done, but it's not my repo so I'd rather ask.

The upgrade shipped in `v3.3`. Of the three items: @kimisgold confirmed above that the close buttons and keyboard-reachable markers were both fixed by it, and the third, announcing the map, is now #77.

The other things raised here also have homes: popup focus was handled on the #71 branch, and shapes are #75.

One loose end: @zerocrates pointed out that Leaflet.markercluster had aria labeling for clusters done upstream but never released, which we could pull in by hand. That still looks outstanding, and #75 looks like the right home for it, since it already covers the accessibility gaps in Leaflet's drawing and clustering plugins.

@zerocrates can this be closed?

**Comment by kimisgold** [comment]
Everything from my initial issue looks resolved from my end, and I think handling the cluster aria labeling in #75 makes sense.

**Comment by jimsafley** [comment]
The cluster ARIA labeling is now tracked in #75, so everything raised here has a home. Closing this out.
